addons/im_livechat/models/discuss_channel.py

```diff
@@ -158,9 +147,10 @@ def _rating_get_parent_field_name(self):
 
     def _email_livechat_transcript(self, email):
         company = self.env.user.company_id
+        # sudo: discuss.channel - required to render the email template
```

Comment:
I wonder if appropriate sudo shouldn't be put in the template instead?
channel should be readable without sudo

Reply:
Indeed, I applied this suggestion and added the required sudo in the template instead.